### ISSUE TITLE
Vagrant builds

### ISSUE THREAD

**Issue by Insoleet** [duniter/sakia#718]
It would be great to have vagrant builds for Sakia like Duniter does (https://github.com/duniter/duniter/tree/master/release)

**Comment by yann1108** [comment]
Inso as-tu trouvé ton "builder" ?

**Comment by Insoleet** [comment]
Non ! A noter qu'il faudrait élargir le profil, même un développeur "Docker" peut le faire a priori.

**Comment by M5oul** [comment]
Oui, c’est ce à quoi je pensais. @sveyret semble avoir le profil pour cette tâche.

**Comment by sveyret** [comment]
Merci pour la proposition. Je la note, mais ne pourrai probablement pas m'y pencher avant un petit moment.

**Comment by yann1108** [comment]
ok, je cherche aussi !

**Comment by heroinedor** [comment]
Ca pourrait m'intéresser. Serait-il possible d'avoir un peu plus de détails sur ce qui est attendu svp ?

**Comment by Insoleet** [comment]
Salut,

L'idée est de réutiliser les scripts développés pour appveyor et travis, mais pour les exécuter dans des VM via l’ordonnanceur Vagrant.

Les yml sont ici : 
https://github.com/duniter/sakia/blob/master/.travis.yml
https://github.com/duniter/sakia/blob/master/appveyor.yml

Et dans le dossier "ci" tu pourras trouver les scripts de compilation.

Du côté de Duniter, il existe un Vagrant file pour windows : https://git.duniter.org/nodes/typescript/duniter/tree/1.6/release/arch/windows

L'objectif est qu'avec Vagrant installé sur son poste, n'importe qui puisse produire une release de Sakia pour tout OS à partir des scripts de build. 

Je ne sais pas si Vagrant supporte OSX (ça m'étonnerait) mais si c'est le cas ça serait niquel.

**Comment by heroinedor** [comment]
Ok, compris le besoin.
Je ne suis pas spécialiste de Vagrant, mais j'ai déjà une petite expérience de Docker.
Je vais voir ce que je peux faire.
